games/naval_battle.c

```diff
@@ -0,0 +1,782 @@
+/**
+ * @file naval_battle.c
```

Comment:
Better not to provide the file name, otherwise you'll need to add the directories where's the file stored.
```suggestion
 * @file
```